recipe/2025-08-recipe-packs.md

```diff
@@ -0,0 +1,379 @@
+# Recipe Packs
+
+* **Author**: `Nithya (@nithyatsu)`
+
+## Overview
+
+Recipes are external infrastructure-as-code (IaC) templates that operators register on a Radius Environment so developers can use them later for provisioning. They provide a mechanism for separation of concerns between developers and operators. 
+
+Today, Radius supports registering recipes individually, either via the `Applications.Core\environments` resource properties or the CLI (`rad recipe register`). For each Radius Environment, platform engineers have to piece together individual Recipes from scratch. Some customers could have 1000s of environments and putting everything together manually for each environment is error prone. Recipes also do not have a lifecycle of their own and can be managed only by managing the environments pointing to them.
+
+This document details the introduction of Recipe Packs as a feature that makes recipe management easier. Recipe Packs are a collection of related recipes that a platform engineer can manage as a single entity. 
+
+## Terms and definitions
+
+| Term     | Definition                                                                                                                                                                                                 |
+| -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Recipe | IaC templates that operators register on a Radius Environment |
+| Recipe Pack| A collection of recipes that can be managed as an entity |
+| RRT | [Radius Resource Type](https://docs.radapp.io/guides/author-apps/custom/overview/)|
+
+## Objectives
+
+> **Issue Reference:** 
+
+### Goals
+
+Provide Recipe Packs as a Radius feature to bundle multiple recipes as a single managable entity into a Radius Environment (e.g., a pack for ACI that includes all necessary recipes).
+
+### Non goals
+
+Recipe Packs would bundle together Recipes, as we understand them today. We do not cover recipe versioning / other recipe specific enhancements in this design. 
+
+
+### User scenarios (optional)
```

Comment:
One more scenario I suggest adding is the ability to share Recipe Packs across environments or even organizations. e.g. Azure or AWS might create Recipes and bundle them into a Recipe Pack that can be used off the shelf, compute platform specific packs (e.g. Kubernetes, ACI) that ship with Radius by default